build.sbt

```diff
@@ -38,3 +39,5 @@ fork in Test := true
 javaOptions ++= Seq("-Xms512M", "-Xmx2048M", "-XX:+CMSClassUnloadingEnabled","-Duser.timezone=GMT")
 
 licenses := Seq("MIT" -> url("http://opensource.org/licenses/MIT"))
+
+updateOptions := updateOptions.value.withLatestSnapshots(false)
```

Comment:
I added this line to be able to work with a local version of the library (using `publishLocal`.

I don't think it hurts if we are not depending on SNAPSHOTS anyway. That being said, maybe I should delete this line.